templates/CRM/common/paymentBlock.tpl

```diff
@@ -85,23 +85,21 @@
       {else}
         {capture assign='preProfileID'}{/capture}
       {/if}
+      {if !array_key_exists('urlPathVar', $form)}{assign var='urlPathVar' value=null}{/if}
```

Comment:
Seems a bit confusing to do this one differently than all the others, why not make this the same as billing_profile_id below?

Reply:
Are either of them elements in the $form array? They both seem like top-level smarty vars. I.e. array_key_exists will always be false. I don't see them available in $form with `{debug}`?

(Note there is also a javascript `$form` to confuse things)

Reply:
Coming back to this it looks like urlPathVar is almost never set `CRM_Event_Form_Task_Register` seems like the only possibly relevant place

![image](https://github.com/civicrm/civicrm-core/assets/336308/cc832aff-7398-474b-8268-fdd1df79d26e)

Reply:
OK - but that is in the chunk of code that calls civicrm/payment/form - ie credit card stuff - which isn't called by The register task based on my debugging

So, from what I can tell that whole `$urlPathVar` does nothing - on the bright side that means that it always being false like @demeritcowboy says would be no change...